mne_features/utils.py

```diff
@@ -18,16 +18,22 @@
 
 
 @nb.jit()
-def triu_idx(n, include_diag=False):
+def idxiter(n, triu=True, include_diag=False):
```

Comment:
you never seem to call this function with triu=False. if so why a parameter that you will not use? btw you could make this function private as it's not public and present in api.rst

Reply:
I'm using `triu=False` in `compute_pow_freq_bands`. It's for this function that I introduced the `triu` optional parameter (to be set to False). I'll make the function private.

Reply:
ok got it. thx